src/sentry/api/bases/organization.py

```diff
@@ -101,6 +101,16 @@ class OrganizationApiKeysPermission(OrganizationPermission):
     }
 
 
+class OrganizationAuthProviderPermission(OrganizationPermission):
+    scope_map = {
+        # Required for members view
+        'GET': ['member:read', 'org:read'],
```

Comment:
This is probably not right, we probably don't want to give member's read access (we'd have to change frontend to allow for 403s).

Reply:
What's up with the comment `Required for members view` in that case ?

Reply:
Refers to frontend - looking for feedback on permissions for this endpoint, should members be allowed access?

Reply:
I'd suggest we start without `member:read` and we can always add it later if it's necessary?